### PR TITLE
Tb user story 16

### PR DESCRIPTION
**Pull request by TravisBorgsmiller**
Added functionality for each user to be able to click a logout link 

Styled nav bar to be able to show cart, and login/logout in the top right part of the nav

Changed the logout link name in a couple places and the according tests

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
```
Comment (KellyIB) [inline db/seeds.rb:6]: these must be references you used, Travis. Can we delete or do you still need them?

Comment (KellyIB) [reply db/seeds.rb:6]: (lines 6 -7)

**Review by KellyIB — APPROVED** [review]
Looks like what might be commented out references in the seeds file.  Do we still need those in there (lines 6-7)? If so, that's fine.  Just have to remember to take them out later.

File: spec/features/users/logout_spec.rb [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+        allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(default_user)
+
+        @meg = Merchant.create(name: "Meg's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
+
+        @tire = @meg.items.create(name: "Gatorskins", description: "They'll never pop!", price: 100, image: "https://www.rei.com/media/4e1f5b05-27ef-4267-bb9a-14e35935f218?size=784x588", inventory: 12)
+
+        visit "/items/#{@tire.id}"
+        click_on "Add To Cart"
+        click_on "Log Out"
```
Comment (cgaddis36) [inline spec/features/users/logout_spec.rb:24]: May want to add in an additional test for the cart incrementation before logging out as well to ensure there are items in the cart before log out

**Review by cgaddis36 — APPROVED** [review]

**Pull request merged**